2169.count-operations-to-obtain-zero.js

```diff
@@ -0,0 +1,25 @@
+/* URL of this problem
+ * https://leetcode.com/problems/count-operations-to-obtain-zero/
+ *
+ * @param {number} num1
+ * @param {number} num2
+ * @return {number}
+ */
+ 
+var countOperations = function(num1, num2) {
+    let operationCount = 0;
+    
+    while (num1 !== 0 && num2 !== 0) {
+        if (num1 >= num2) {
+            num1 -= num2;
+        } else {
+            num2 -= num1;
+        }
```

Comment:
```suggestion
        num1 >= num2 ? num1 -= num2 : num2 -= num1;
```
You can also write like this at here